src/main/java/com/icthh/xm/tmf/ms/customer/domain/CustomerCharacteristicEntity.java

```diff
@@ -1,17 +1,14 @@
 package com.icthh.xm.tmf.ms.customer.domain;
 
-import static javax.persistence.GenerationType.SEQUENCE;
-
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.Table;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import javax.persistence.*;
```

Comment:
Wildcard imports should not be used according to our checkstyle

Reply:
Fixed